rust_src/src/dired.rs

```diff
@@ -9,7 +9,15 @@ use remacs_sys::build_string;
 use remacs_sys::globals;
 
 use lisp::{defsubr, LispObject};
-use lists::list;
+use lists::{car, list};
+use strings::string_lessp;
+
+/// Return t if first arg file attributes list is less than second.
+/// Comparison is in lexicographic order and case is significant.
+#[lisp_fn(min = "2")]
```

Comment:
You only need the "min" if it is possible to call this function with a different number of arguments than specified to the Rust function. Otherwise the compilation will set this up correctly.

Reply:
Good catch!